src/Config/IgnitionConfig.php

```diff
@@ -29,6 +32,15 @@ public function setOption(string $name, string $value): self
         return $this;
     }
 
+    private function initConfigManager(): ConfigManager
+    {
+        if (! function_exists('app')) {
+            return new FileConfigManager();
+        }
+
+        return app(ConfigManager::class);
```

Comment:
Interesting approach! :D

Reply:
Frankly speaking, this approach is a little bit dangerous, because it leads to an exception if there is no bound instance in the container (according to PSR-11 and [according to the documentation](https://laravel.com/docs/9.x/container#psr-11) Laravel follows it). In my view this is the case where `fail fast` looks better than `fail soft`.